src/notification.rs

```diff
@@ -330,6 +343,44 @@ impl Notification {
         Ok(NotificationHandle::new(self.clone()))
     }
 
+     /// Sends Notification to NSUserNotificationCenter.
+    ///
+    /// Returns an `Ok` no matter what, since there is currently no way of telling the success of
+    /// the notification.
+    #[cfg(target_os = "windows")]
+    pub fn show(&self) -> Result<()> {
+        let sound_name = self.sound_name.clone();
+        let sound = match sound_name {
+            Some(chosen_sound_name) => winrt_notification::Sound::from_str(&chosen_sound_name).ok(),
+            None => None
+        };
+
+        let duration = match self.timeout {
+            Timeout::Default => winrt_notification::Duration::Short,
+            Timeout::Never => winrt_notification::Duration::Long,
+            Timeout::Milliseconds(t) => if t >= 25000 {
```

Comment:
where do you take these 25s from? is that documented anywhere?

Reply:
https://allenbenz.github.io/winrt-notification/0_2_0/winrt_notification/enum.Duration.html